src/Core/src/Handlers/SearchBar/SearchBarHandler.cs

```diff
@@ -17,7 +17,7 @@ public partial class SearchBarHandler : ISearchBarHandler
 	{
 		public static IPropertyMapper<ISearchBar, ISearchBarHandler> Mapper = new PropertyMapper<ISearchBar, ISearchBarHandler>(ViewHandler.ViewMapper)
 		{
-#if __IOS__
+#if ANDROID || __IOS__
```

Comment:
I'm a bit confused why we have this ifdef here at all. @rmarinho asked about Windows and I just checked - the IsEnabled property is wired up properly and does work as anticipated! (Regardless of this compiler directive, interestingly - not really sure why)

In any case, I think this ifdef can be removed, and have tested that Windows and Android compile and behave as expected with its removal

```suggestion
```

Reply:
Yeah, is working on Windows but without this compilation directive in the SearchBar is not working on Android and iOS.

Android (without ifdef)
<img width="1152" alt="Captura de pantalla 2023-05-10 a las 16 16 49" src="https://github.com/dotnet/maui/assets/6755973/a5bda60a-050f-4466-a804-4d86fcf6c330">

iOS (without ifdef)
<img width="980" alt="Captura de pantalla 2023-05-10 a las 16 11 40" src="https://github.com/dotnet/maui/assets/6755973/d343fd31-ff8b-45bb-9d9d-be7f83e8988b">

Reply:
Without the compilation directive, the `IsEnabled `property works on Windows and not on Android and iOS. I can investigate why, but without other changes, while in Windows it is not necessary, it is not the case with Android and iOS.

Reply:
hmm without the directive, I was able to test that it works on Windows, Android, and iOS for me. happy to hop on a call and show you if that helps clarify things!

Reply:
Ok, in that case the PR is not necessary.